internal/export/ingest_pipelines.go

```diff
@@ -96,23 +97,27 @@ func writePipelineToFile(pipeline ingest.RemotePipeline, writeLocation PipelineW
 		return fmt.Errorf("unmarshalling ingest pipeline failed (ID: %s): %w", pipeline.Name(), err)
 	}
 
-	yamlBytes, err := yaml.Marshal(jsonPipeline)
+	delete(jsonPipeline, "_meta")
+	delete(jsonPipeline, "version")
+
+	var documentBytes bytes.Buffer
+	// requirement: https://github.com/elastic/package-spec/pull/54
+	documentBytes.WriteString("---\n")
+	yamlEncoder := yaml.NewEncoder(&documentBytes)
+	yamlEncoder.SetIndent(2)
+	err = yamlEncoder.Encode(jsonPipeline)
```

Comment:
Btw, we have a `formatter` package with shared formatters for JSON and YAML. They do basically what we are doing here, with the exception of filtering out fields.

Maybe we can add a method `FormatWithFilter(content []byte, filter func(map[string]any))`

But current approach is fine too, so as you prefer.